server/models/model/User.py

```diff
@@ -0,0 +1,62 @@
+from sqlalchemy.ext.hybrid import hybrid_property
+from flask_login import UserMixin
+
+from server.constants import VALID_ROLES, STUDENT_ROLE
+from server.utils import humanize_name
+from server.models.db import db, Model
+from server.extensions import cache
+
+from server.models import GradingTask, ModelProxy
+
+class User(Model, UserMixin):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(255))
+    email = db.Column(db.String(255), unique=True, nullable=False, index=True)
+    is_admin = db.Column(db.Boolean(), default=False)
+
+    export_items = ('email', 'name')
+
+    def __repr__(self):
+        return '<User {0}>'.format(self.email)
+
+    def enrollments(self, roles=None):
+        Enrollment = ModelProxy.Enrollment
```

Comment:
Same: make `enrollments` (and maybe `is_enrolled`) static methods? Or do a `filter` on `self.participations` like `is_enrolled` currently does